### DISCUSSION TITLE
🎉Issue party?

### DISCUSSION THREAD

**Discussion by shykes** [dagger/dagger#3594] [Ideas]
## Problem

We are being sloppy about creating issues and keeping them up to date. We frequently have communication breakdowns where impromptu discord discussions (good) result in PRs (good) without clear context for anyone who wasn't in the discord thread (bad).

## Solution

We should develop a habit that "if there's no issue, it doesn't exist".

To help us switch to this mindset, I propose organizing a **one-hour issue party**.

* The *issue party* is a one-hour zoom call open to all contributors and maintainers
* The goal of this call is to create and update as many issues are possible
* The call MC starts and curates a google doc which everyone on the call can edit
* Everyone on the call jogs their memory and adds to the doc their list of things that need an issue
* Then the work of searching for an issue, updating it or creating it as needed, is dispatched to all participants
* At the end of the hour, participants count their score; one point per issue created, updated or closed
* The next issue party tries to beat the score of the first :)

Thoughts?

**Comment by grouville** [comment]
Excellent idea. When is the next one ? 💥

**Reply by shykes** [reply]
cc @mircubed 😇

**Reply by mircubed** [reply]
Happy to organize it! I'll look at some options and send something out soon.

**Comment by vikram-dagger** [comment]
+1

**Comment by mircubed** [comment]
I love this idea!

**Comment by dolanor** [comment]
Having spent a lot of time trying to build context out of discord conversation and issues, I can't agree more with that idea :)

**Comment by marcosnils** [comment]
Let's do it!

<img src="https://media.tenor.com/kxzr3-r6XoIAAAAM/lets-get-this-party-started-yeah.gif"/>

**Comment by mircubed** [comment]
Ok, working on scheduling this shortly. Will add a link to the Discord event to this post once it is scheduled.
